server/repository/src/db_diesel/system_log_row.rs

```diff
@@ -0,0 +1,108 @@
+use crate::{ChangeLogInsertRow, ChangelogRepository, ChangelogTableName, RowActionType, Upsert};
+use crate::{RepositoryError, StorageConnection};
+
+use chrono::NaiveDateTime;
+use diesel::prelude::*;
+use diesel_derive_enum::DbEnum;
+use serde::{Deserialize, Serialize};
+
+table! {
+    system_log (id) {
+        id -> Text,
+        #[sql_name = "type"] type_ -> crate::db_diesel::system_log_row::SystemLogTypeMapping,
+        sync_site_id -> Nullable<Integer>,
+        datetime -> Timestamp,
+        message -> Nullable<Text>,
+        is_error -> Bool,
```

Comment:
Thought this would be useful for filtering when/if we add other kinds of events into this log...